src/AFL-1.2.xml

```diff
@@ -0,0 +1,78 @@
+<SPDX name="Academic Free License v1.2" identifier="AFL-1.2" osi-approved="true">
+  <urls>
+    <url>http://opensource.linux-mirror.org/licenses/afl-1.2.txt</url>
+  </urls>
+  <notes>This license has been superseded by later versions.</notes>
+  <license>
+    <title>
+      <p>Academic Free License 
+        <br/>Version 1.2 
+      </p>
+    </title>
+    <optional>
+      <p>This Academic Free License applies to any original work of authorship (the "Original Work") whose owner
+         (the "Licensor") has placed the 
+        <br/>following notice immediately following the copyright notice for the Original Work: 
+      </p>
+      <p>Licensed under the Academic Free License version 1.2</p>
+    </optional>
+    <body>
+      <p>Grant of License. Licensor hereby grants to any person obtaining a copy of the Original Work ("You") a
+         world-wide, royalty-free, non-exclusive, perpetual, non-sublicenseable license (1) to use, copy,
+         modify, merge, publish, perform, distribute and/or sell copies of the Original Work and derivative
+         works thereof, and (2) under patent claims owned or controlled by the Licensor that are embodied in
+         the Original Work as furnished by the Licensor, to make, use, sell and offer for sale the Original
+         Work and derivative works thereof, subject to the 
+        <br/>following conditions. 
```

Comment:
Yes, I believe we can & should remove be tags, replacing them with p tags
where necessary (though I wouldn't say it's necessary here).